console_test.go

```diff
@@ -15,48 +15,71 @@
 package main
 
 import (
+	"io/ioutil"
 	"os"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestConsoleFromFile(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
Hmm, maybe we should be using this in other places too, particularly since it simplifies the calling ;)